packages/global-styles-ui/src/font-library/lib/inflate.js

```diff
@@ -30,25 +30,8 @@
 
 /* eslint eslint-comments/no-unlimited-disable: 0 */
 /* eslint-disable */
-/* pako 1.0.10 nodeca/pako */ ( function ( f ) {
-	if ( typeof exports === 'object' && typeof module !== 'undefined' ) {
-		module.exports = f();
-	} else if ( typeof define === 'function' && define.amd ) {
-		define( [], f );
-	} else {
-		var g;
-		if ( typeof window !== 'undefined' ) {
-			g = window;
-		} else if ( typeof global !== 'undefined' ) {
-			g = global;
-		} else if ( typeof self !== 'undefined' ) {
-			g = self;
-		} else {
-			g = this;
-		}
-		g.pako = f();
-	}
-} )( function () {
+/* pako 1.0.10 nodeca/pako */
+export default ( function () {
```

Comment:
This converts the 3rd party vendored module from UMD to ESM. Turns out to be a very simple change.